orchagent/bfdorch.cpp

```diff
@@ -298,6 +352,10 @@ bool BfdOrch::create_bfd_session(const string& key, const vector<FieldValueTuple
         {
             tos = to_uint<uint8_t>(value);
         }
+        else if (fvField(i) == "tsa_shutdown")
```

Comment:
I have added a comment in the code and I have changed the name of the field to shutdown_bfd_during_tsa. The BFD tsa shutdown is handled by the bfd cache in the caller function. This block is added to avoid getting the unknown parameter syslog.